apps/condo/domains/news/components/NewsForm/BaseNewsForm.tsx

```diff
@@ -107,10 +108,19 @@ export const StyledTabs = styled(Tabs)`
         }
     }
 `
-const COUNTER_COL_STYLE: React.CSSProperties = {
+const COUNTER_TITLE_STYLE: React.CSSProperties = {
+    top: '123px',
+    position: 'absolute',
+    right: 0,
+    margin: '12px',
+    padding: '2px 10px',
+    backgroundColor: 'black',
```

Comment:
it's better to import a collection of colors. It seems you can take here import { colors } from '@open-condo/ui/dist/colors'